launchpod/angular-launchpod/src/app/episode-choice/episode-choice.component.html

```diff
@@ -0,0 +1,66 @@
+<body>
+<h2>Choose An Option for Adding Audio to Your Episode</h2>
+
+<div>
+  <mat-card>
+    <br />
+    <mat-card-header>
+      <mat-card-title>Link</mat-card-title>
+    </mat-card-header>
+    <br />
+    <mat-card-content>
+      <p>
+        Paste a link to a publicly accessible
+        <br />
+        MP3 file.
+      </p>
+      <mat-icon>link</mat-icon>
+    </mat-card-content>
+    <br />
+    <mat-card-actions>
+      <button mat-raised-button color="primary" routerLink='/episode-link-form'>MP3 LINK</button>
+    </mat-card-actions>
+    <br />
+  </mat-card>
+  <mat-card>
+    <br />
+    <mat-card-header>
+      <mat-card-title>Upload</mat-card-title>
+    </mat-card-header>
+    <br />
+    <mat-card-content>
+      <p>
+        Upload a local MP3 file 
+        <br />
+        from your computer.
```

Comment:
Why add a line break instead of having the text wrap itself? This should happen automatically if there is a size given to the container: https://codepen.io/bramanudom/pen/QWNLaxb

Reply:
done.